### DISCUSSION TITLE
Position of the Outline Pane

### DISCUSSION THREAD

**Discussion by sk-akiyama** [coteditor/CotEditor#1721] [Ideas]
Thank you for reviewing this discussion.
I always enjoy using CotEditor!

Regarding the position of the outline pane, would it be possible to move it to the left side?
I believe this would be more natural and user-friendly in terms of the UI!

If possible, I would appreciate your consideration.
I will continue to support the development of CotEditor.

<img width="720" alt="スクリーンショット 2024-09-22 午前11 31 15" src="https://github.com/user-attachments/assets/395c89f5-ae2d-4ed4-b1e6-e3962613b00f">

**Comment by 1024jp** [comment]
Thank you for the idea.
However, this proposal is contrary to the role of the window panes. Each pane has its own role.
While the inspector, the right pane, provides details of the current document, the sidebar, the left pane, provides the information on the folder level.
So the outline list should be in the inspector.

**Comment by sk-akiyama** [comment]
Thank you for your reply!
I see… each pane has its own role.

I’ll do my best to get used to things.

Thank you!
